code/modules/ghostroles/spawner/human/responseteams/unathi/izweski.dm

```diff
@@ -0,0 +1,37 @@
+/datum/ghostspawner/human/ert/izweski
```

Comment:
Only the leadership role in a species restricted ERT may be whitelisted.
All the other roles must be accessible without a whitelist.